common/v2/features/BuyAndExchange/ZeroEx/ZeroEx.tsx

```diff
@@ -1,59 +1,46 @@
 import React, { Component } from 'react';
 
-import { isDesktop } from 'v2/utils';
-import { replaceZeroExContainer } from './helpers';
-import {
-  ZEROEX_CONTAINER_ID,
-  ZEROEX_RELAYER,
-  ZEROEX_FEE_ADDRESS,
-  ZEROEX_FEE_PERCENTAGE
-} from './constants';
+// import { isDesktop } from 'v2/utils';
+// import { replaceZeroExContainer } from './helpers';
+// import {
+//   ZEROEX_CONTAINER_ID,
+//   ZEROEX_RELAYER,
+//   ZEROEX_FEE_ADDRESS,
+//   ZEROEX_FEE_PERCENTAGE
+// } from './constants';
```

Comment:
Feel free to remove the commented code.
As long as the commit message and the description are sufficient it will be easy to revert the commit if we ever decide to reactivate.

Reply:
Removed in `fcba8fa`